firebase.json

```diff
@@ -33,5 +31,15 @@
     "auth": {
       "port": 9099
     }
+  },
+  "hosting": {
+    "public": "public",
+    "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
+    "rewrites": [
+      {
+        "source": "**",
```

Comment:
If we build a web UI for managing API tokens, etc. where would it live?  I could imagine either:

1) We move the API to a sub-path, like https://share.actnowcoalition.org/api/ 
2) We put the admin UI on a sub-path, like https://share.actnowcoalition.org/admin/
3) We create separate subdomains (but I am not 100% sure if this would require setting up multiple firebase projects, etc. which would be kind of annoying).

I don't have strong feelings, but we might want to give it a little thought to make sure we are set up for this in the future.

Reply:
Good question,

Yeah, I think we could maybe use cascading rewrites to route the web UI to https://share.actnowcoalition.org/admin/, while keeping the API as it is with something like:

```JSONc
"rewrites": [
  {
    "source": "admin/**",
    "destination": "public/admin/" // this probably isn't the right syntax
  },
  {
   "source": "**",
    "function": "api"
  }
]
```

which would catch all the web UI pages, then fallback to the API. 

Also, I haven't looked into it, but maybe using targets could be appropriate here, although that seems like maybe too much work for this case https://firebase.google.com/docs/cli/targets. 

Would it feel satisfactory to leave this as-is, with the intention of putting the web UI on a sub-path when it's created?

Reply:
Yep, I think so!